pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -227,6 +228,15 @@ func (c *Cluster) makeSetServerAddrInitContainer(mgrConfig *mgrConfig, mgrModule
 			c.cephMgrOrchestratorModuleEnvs()...,
 		),
 		Resources: c.resources,
+		LivenessProbe: &v1.Probe{
```

Comment:
The LivenessProbe should go on the main container (around line 245) instead of an init container. The CI is showing this error in the operator logs:
```
2019-05-15 05:58:06.000977 E | op-cluster: failed to create cluster in namespace block-k8s-ns. failed to start the ceph mgr. failed to create mgr deployment rook-ceph-mgr-a. Deployment.apps "rook-ceph-mgr-a" is invalid: [spec.template.spec.initContainers[0].livenessProbe: Invalid value: core.Probe{Handler:core.Handler{Exec:(*core.ExecAction)(nil), HTTPGet:(*core.HTTPGetAction)(0xc003788fc0), TCPSocket:(*core.TCPSocketAction)(nil)}, InitialDelaySeconds:60, TimeoutSeconds:1, PeriodSeconds:10, SuccessThreshold:1, FailureThreshold:3}: 
must not be set for init containers, spec.template.spec.initContainers[1].livenessProbe: Invalid value: core.Probe{Handler:core.Handler{Exec:(*core.ExecAction)(nil), HTTPGet:(*core.HTTPGetAction)(0xc003789030), TCPSocket:(*core.TCPSocketAction)(nil)}, InitialDelaySeconds:60, TimeoutSeconds:1, PeriodSeconds:10, SuccessThreshold:1, FailureThreshold:3}: must not be set for init containers]
```

This likely wasn't seen in your local testing since this init container is not always created.

Reply:
ouch... thanks for spotting that!